spinalcordtoolbox/utils/sys.py

```diff
@@ -88,6 +88,16 @@ def set_loglevel(verbose):
         pass
 
 
+def strip_py(fname):
+    """
+    Strip ".py" suffix from the string `fname`, if present.
+
+    :param fname: str: a filename
+    :return: a stripped copy of `fname`
+    """
+    return fname[:-3] if fname.endswith('.py') else fname
```

Comment:
How would you feel about adopting the more general version of the function (`str.removesuffix`), since the source code is provided in the [PEP proposal](https://www.python.org/dev/peps/pep-0616/)?

```python
def removesuffix(self: str, suffix: str, /) -> str:
    """
    Source: https://www.python.org/dev/peps/pep-0616/

    TODO: Replace with built-in str.removesuffix method after upgrading to Python 3.9
    """
    # suffix='' should not call self[:-0].
    if suffix and self.endswith(suffix):
        return self[:-len(suffix)]
    else:
        return self[:]
```

Reply:
Yes!!! I was initially so surprised that this _still_ didn't seem to exist as a built-in, but I think I was just looking at old docs. And your doc-comment, with the official source and the TODO to eventually get rid of the code, are perfect. Thank you! I'll use this version instead.